.github/pull_request_template.md

```diff
@@ -27,3 +31,41 @@ Major changes require a number of considerations including impact on:
 * Elasticsearch Service (https://cloud.elastic.co)
 * Elastic Cloud Enterprise (https://www.elastic.co/products/ece)
 -->
+
+## Checklist
+<!--
+Add a checklist of things that are required to be reviewed in order to have the PR approved
+List here all the items you have verified BEFORE sending this PR. Please DO NOT remove any item, striking through those that do not apply.
+
+List here all the items you have verified BEFORE sending this PR. Please DO NOT remove any item, striking through those that do not apply. (Just in case, strikethrough uses two tildes. ~~Scratch this.~~)
+-->
+- [ ] I have signed the [Contributor License Agreement](https://www.elastic.co/contributor-agreement/). 
+- [ ] My code follows the style guidelines of this project
+- [ ] I have rebased my changes on top of the latest master branch
```

Comment:
We do use a linter in the first CI stage. Should we add a checkbox for ensuring the `check-full` passes?

Reply:
This template is filled out before reaching the CI. I'd add here actions the sender must do before submitting the PR. Maybe we could explain that in the `My code follows the style guidelines...`. If so, what info would you add there?

Reply:
Agreed. I was referring to running `make check-full` before creating the PR. So maybe something along the lines of:
_My code follows the style guidelines of this project (run `make check-full` for static code checks and linting)._